### PR TITLE
[DO NOT MERGE] Enable console log for cache in production

### PR DESCRIPTION
**Pull request by iAdramelk**
Brunch for testing cache on heroku deploy, do not merge.

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
> [DO NOT MERGE]

You can use GH Draft PR to prevent them from being merged accidentally 🙂

File: pages/api/comments.js [hunk 1]
```diff
@@ -36,13 +34,13 @@ const getCommentCount = async (req, res) => {
   }
 
   if (cache.get(url) !== undefined) {
-    if (dev) console.log(`Using cache for ${url}`)
+    console.log(`Using cache for ${url}`)
```
Comment (jorgeorpinel) [inline pages/api/comments.js]: `console.debug()`?

**Comment by shcheklein** [comment]
@iAdramelk can we close this now?

**Comment by iAdramelk** [comment]
Closed it

**Pull request closed without merging**